diagrams/onprem/network.py

```diff
@@ -54,6 +54,10 @@ class Nginx(_Network):
 
 class Ocelot(_Network):
     _icon = "ocelot.png"
+    
+    
+class Openservicemesh(_Network):
```

Comment:
The class name for open service mesh should be `class OpenServiceMesh`. This is how `autogen` script names the class:

1. Split the filename by `-` delimiter and join each element after making each one as title-case. (`open-service-mesh.png` will be translated to `class OpenSerivceMesh`)
2. If you register the component name to `TITLE_WORDS` dict, it will make the class named as a registered name regardless of filename.

So you should make the filename as `open-service-mesh.png` or register the open service mesh name to `TITLE_WORDS` to make the title cased class name, the `class OpenServiceMesh`.

You can find `TITLE_WORDS` dict in [here](https://github.com/mingrammer/diagrams/pull/260/files#diff-fdf313ed2303059c44a9ebfffbef8eebR74-R87).